rvic/core/config.py

```diff
@@ -84,7 +84,7 @@ def config_type(value):
 def isfloat(x):
     """Test of value is a float"""
```

Comment:
if - yes. I don't think it is worth adding a dependency for a simple function like this.